src/benchmark/runner.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  File,
+  startTests,
+  Suite,
+  Task,
+  VitestRunner,
+  VitestRunnerConfig,
+  VitestRunnerImportSource,
+} from "@vitest/runner";
+import {Benchmark, BenchmarkOpts, BenchmarkResults} from "../types.js";
+import {BenchmarkReporter} from "./reporter.js";
+import {store} from "./globalState.js";
+
+export class BenchmarkRunner implements VitestRunner {
+  readonly config: VitestRunnerConfig;
+  readonly reporter: BenchmarkReporter;
+  readonly prevBench: Benchmark | null;
+  readonly benchmarkOpts: BenchmarkOpts;
+
+  constructor({prevBench, benchmarkOpts}: {prevBench: Benchmark | null; benchmarkOpts: BenchmarkOpts}) {
+    this.config = {
+      root: "",
+      sequence: {seed: 1234, hooks: "list", setupFiles: "list"},
```

Comment:
interesting seed.  what is that used for?

Reply:
To sort the test files, can be any used anything. As long as it's the same tests run in same sequence.